logs/log.txt

```diff
@@ -2634,3 +2634,399 @@ java.lang.RuntimeException: TESTE
+	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:220)
+	at org.junit.platform.launcher.core.DefaultLauncher.lambda$execute$6(DefaultLauncher.java:188)
+	at org.junit.platform.launcher.core.DefaultLauncher.withInterceptedStreams(DefaultLauncher.java:202)
+	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:181)
+	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:128)
+	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invokeAllTests(JUnitPlatformProvider.java:150)
+	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invoke(JUnitPlatformProvider.java:124)
+	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
+	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
+	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
+	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
+{ "timestamp": 02-02-2022 09:49:19.511, "message": Offers will be listed in pageable form., "logger_name": com.product.service.ProductService, "thread_name": [main], "level": INFO  }
+{ "timestamp": 02-02-2022 09:49:19.517, "message": id [1] will be for search for!, "logger_name": com.product.service.ProductService, "thread_name": [main], "level": INFO  }
+{ "timestamp": 02-02-2022 09:49:19.518, "message": The return [Product(id=1, name=Dorflex, description=Relachante muscular para dores abnorminais, price=5.0, theamount=5.0, category=1, classification=1)], "logger_name": com.product.service.ProductService, "thread_name": [main], "level": WARN  }
+{ "timestamp": 02-02-2022 09:49:19.520, "message": The id delete [1], "logger_name": com.product.service.ProductService, "thread_name": [main], "level": INFO  }
+{ "timestamp": 02-02-2022 09:49:19.521, "message": The product delete [Product(id=1, name=Dorflex, description=Relachante muscular para dores abnorminais, price=5.0, theamount=5.0, category=1, classification=1)], "logger_name": com.product.service.ProductService, "thread_name": [main], "level": WARN  }
+{ "timestamp": 02-02-2022 09:49:19.614, "message": Successfully saved!, "logger_name": com.product.service.ProductService, "thread_name": [main], "level": INFO  }
+{ "timestamp": 02-02-2022 09:49:19.617, "message": id [1] will be for updated by, "logger_name": com.product.service.ProductService, "thread_name": [main], "level": INFO  }
+{ "timestamp": 02-02-2022 09:49:19.617, "message": The updated [Product(id=1, name=Dorflex, description=Relachante muscular para dores abnorminais, price=5.0, theamount=5.0, category=1, classification=1)], "logger_name": com.product.service.ProductService, "thread_name": [main], "level": WARN  }
+{ "timestamp": 02-02-2022 09:49:19.621, "message": id [3] will be for search for!, "logger_name": com.product.service.ProductService, "thread_name": [main], "level": INFO  }
```

Comment:
esse arquivo enorme é necessário comitar pro projeto rodar?

Reply:
Esse arquivo é aonde eu guardo os logs.....

Reply:
num projeto de produção os logs ficam no splunk ou no grafana, não é necessario comitar logs, eles aumentam a quantidade de coisa pra fz o review. sendo assim, seria bom removê-lo n?

Reply:
Tudo bem então....
Quando eu for comitar novamente vou manda só o projeto em si....

Reply:
por favor remova esse arquivo e pense numa maneira de não mais subi-lo